MobiusCore/Test/EffectHandlers/EffectRouterDSLTests.swift

```diff
@@ -253,7 +253,6 @@ class EffectRouterDSLTests: QuickSpec {
                     }
 
                 dslHandler.accept(.effect1)
-                dslHandler.accept(.effect2)
```

Comment:
This is invalid and caused an error, but it wasn’t visible when running all test cases because some other test’s error handler was left in place